README.md

```diff
@@ -1329,6 +1329,8 @@ features: {
 
 - **features.showPasswordToggleOnSignInPage** - End users can now toggle visibility of their password on the Okta Sign-In page, allowing end users to check their password before they click Sign In. This helps prevent account lock outs caused by end users exceeding your org's permitted number of failed sign-in attempts. Note that passwords are visible for 30 seconds and then hidden automatically. Defaults to `false`.
 
+- **features.useDeviceFingerprintForSecurityImage** - Uses device fingerprint to retrieve security image. Useful in cases where cookies are cleared off when the app is closed (e.g. embedded browsers). Defaults to `false`.
```

Comment:
should we mention that this feature depends on `deviceFingerprinting`? (which I don't think is documented here...)

cc: @robertdamphousse-okta we are adding a new feature to the list